website/blog/2024-02-12-How-a-POC-became-a-production-ready-Hudi-data-lakehouse-through-close-team-collaboration.mdx

```diff
@@ -0,0 +1,16 @@
+---
+title: "How a POC became a production-ready Hudi data lakehouse through close team collaboration"
+excerpt: "How a POC became a production-ready Hudi data lakehouse through close team collaboration"
+author: leboncoin tech blog
+category: blog
+image: /assets/images/blog/2024-02-12-How-a-POC-became-a-production-ready-Hudi-data-lakehouse-through-close-team-collaboration.png
+tags:
+- blog
+- apache hudi
+- medium
```

Comment:
Add tags such as `deletes` `gdpr deletion` `upserts` ?

Reply:
i put the singular version